students/stachowska_agata/lesson_06_dicts_tuples_sets_args_kwargs/uppercase.py

```diff
@@ -0,0 +1,10 @@
+def capitalize(lower_case_word):
+    words = lower_case_word.split(' ')
+    new_sentence = ''
+    for word in words:
+        new_sentence += chr(ord(word[0]) - 32) + word[1:] + ' '
```

Comment:
There is dedicated function for it
word.title()
or
word[0].upper() + word[1:]

You might also consider to create a list and then join the elements
```
new_sentence = []
 for word in words:
    new_sentence.append(word.title())
return " ".join(new_sentence)